content/en/blog/2023-03-23-rustls-new-features.md

```diff
@@ -0,0 +1,24 @@
+---
+author: Josh Aas
+date: 2023-03-20T00:00:00Z
+slug: rustls-new-features
+title: "Rustls 0.21.0 Released With Exciting New Features"
+excerpt: "We’re incredibly excited about the latest release of Rustls, a memory safe TLS implementation."
+---
+
+We're incredibly excited about the latest release of [Rustls](https://github.com/rustls/rustls), a memory safe TLS implementation. This release has two major new features and a number of other improvements.
+
+The first big feature is support for TLS certificates containing IP addresses. Rustls can now be used to set up TLS connections addressed by IP rather than a domain name. This has been the most heavily requested feature for quite a while now and it's great to have it completed.
```

Comment:
idea from the sidelines: it might be nice to give some indication of _why_ this is a heavily requested feature.

AIUI there are two significant use-cases:

1) For DNS over HTTPS or DNS over TLS where you need an IP address for the server to avoid circular dependency on name resolution
2) For Kubernetes environments where pods are often addressed by IP and may not have a domain name.

```suggestion
The first big feature is support for TLS certificates containing IP addresses. Rustls can now be used to set up TLS connections addressed by IP rather than a domain name. This has been the most heavily requested feature for quite a while now and it's great to have it completed. Enabling secure connections to IP addresses will help adoption of Rustls for securing DNS over TLS/HTTPS and in cloud deployments without stable domain names.
```